modules/importing-manifest-list-through-imagestreamimport.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+// * assembly/openshift_images/managing-image-streams.adoc
+
+:_content-type: PROCEDURE
+[id="importing-manifest-list-through-imagestreamimport_{context}"]
+= Importing a manifest list through ImageStreamImport
+
+The `ImageStreamImport` resource provides users a way to find and import image manifests from other container image registries into the cluster. Individual images or an entire image repository can be imported. 
+
+Use the following procedure to import a manifest list through the `ImageStreamImport` object with the `importMode` value. 
+
+.Procedure
+
+. Create an ImageStreamImport YAML file and set the `importMode` to `PreserveOriginal` on the tags that you will import as a manifest list:
++
+[source,yaml]
+----
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStreamImport
+metadata:
+  name: app
+  namespace: myapp
+spec:
+  import: true
+  images:
+  - from:
+      kind: DockerImage
+      name: <registry>/<user_name>/<image_name>
+    to:
+      name: latest
+    referencePolicy:
+      type: Source
+    importPolicy:
+      importMode: "PreserveOriginal"
```

Comment:
Should we mention another value "Legacy" for importMode here to explain the importMode usage? @dmage @flavianmissi

Reply:
That wouldn't hurt. "Legacy" is the default value, if nothing else we could just say that...?

Reply:
@flavianmissi @xiuwang PTAL at the table I added to the section below. Thanks.